docs/assets/index.scss

```diff
@@ -1,6 +1,6 @@
 $generate-doc-comments: true;
 
-@import "./src/styles";
+@import "//src/styles";
```

Comment:
I'm not familiar with SASS' latest changes. Is this the way to do it now?

Reply:
Apparently. This is the solution that worked after a quick research.